renderer/sources.gni

```diff
@@ -4,10 +4,21 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 brave_chrome_renderer_sources = [
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_render_frame_observer.cc",
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_render_frame_observer.h",
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_handler.cc",
+  "//brave/components/cosmetic_filters/content/renderer/cosmetic_filters_js_handler.h",
   "//brave/renderer/brave_content_renderer_client.cc",
   "//brave/renderer/brave_content_renderer_client.h",
 ]
+
 brave_chrome_renderer_public_deps = [
+  "//base",
```

Comment:
all of these are already included into deps in `src/chrome/renderer/BUILD.gn`, so not sure if we need them. Maybe @bridiver can comment